### PR TITLE
f-checkout@v0.173.0  Refactor checkout mocks.

### PR DESCRIPTION
**Pull request by BanksySan**
Refactoring storybook tests to make them nice.

### CODE REVIEW

File: packages/components/organisms/f-checkout/src/demo/checkoutMock.js [hunk 1]
```diff
@@ -1,145 +1,46 @@
...
     }
+
+    if (mockedRequest.responseStatus === httpStatusCodes.noResponse) {
+        methodMock.timeout();
+    } else {
+        methodMock.reply(mockedRequest.responseStatus, mockedRequest.payload);
+    }
+}
+
+function setupCheckoutMethods () {
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-checkout/src/demo/checkoutMock.js:31]: I think this name clashes with the one above `setupCheckoutMethod` can we change it?

File: packages/components/organisms/f-checkout/src/demo/checkoutMock.js [hunk 2]
```diff
@@ -1,145 +1,46 @@
...
+
+    for (let i = 0; i < requests.length; i++) {
+        const [, value] = requests[i];
+
+        setupCheckoutMethod(value);
+    }
+    mock.onAny().passThrough();
+}
+
+export default setupCheckoutMethods;
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-checkout/src/demo/checkoutMock.js:42]: Does this need to be exported?

File: packages/components/organisms/f-checkout/src/demo/checkoutMock.js [hunk 3]
```diff
@@ -1,145 +1,46 @@
...
+
+export {
+    mockedRequests
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-checkout/src/demo/checkoutMock.js]: I may have missed something but how comes we're exporting something that's already been imported?

File: packages/components/organisms/f-checkout/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v0.173.0
+------------------------------
+*August 16, 2021*
+
+### Added
+- New file responsible for the mack response data.
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-checkout/CHANGELOG.md]: `mock`

File: packages/components/organisms/f-checkout/CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,18 @@
...
+
+### Changed
+
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-checkout/CHANGELOG.md]: Minor, could you remove the line break please. :)

File: packages/components/organisms/f-checkout/src/demo/checkoutMock.js (hunk 1 above)
Comment (BanksySan) [reply packages/components/organisms/f-checkout/src/demo/checkoutMock.js:31]: I think they're named correctly, `setupCheckoutMethod()` sets up a single method, `setupCheckoutMethods()` sets up many methods.

File: packages/components/organisms/f-checkout/src/demo/checkoutMock.js (hunk 2 above)
Comment (BanksySan) [reply packages/components/organisms/f-checkout/src/demo/checkoutMock.js:42]: It's called in `Checkout.stories.js` line 25.

File: packages/components/organisms/f-checkout/src/demo/checkoutMock.js (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-checkout/src/demo/checkoutMock.js:31]: Yeah i just mean that at first it looked like a single method calling itself until i saw the `s`.

File: packages/components/organisms/f-checkout/src/demo/checkoutMock.js (hunk 3 above)
Comment (BanksySan) [reply packages/components/organisms/f-checkout/src/demo/checkoutMock.js]: Good point.  I've corrected it.

**Review by kevinrodrigues — DISMISSED** [review]

**Review by charliearlie — DISMISSED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Review by charliearlie — APPROVED** [review]

**Pull request merged**